cmd/av/stack_tree.go

```diff
@@ -26,32 +30,49 @@ var stackTreeCmd = &cobra.Command{
 			return err
 		}
 
+		currentBranch, err := repo.CurrentBranchName()
+		if err != nil {
+			return err
+		}
+
 		// TODO[polish]:
 		// 		We should show information about whether or not each branch is
 		//     	up-to-date with its stack parent as well as whether or not the
 		//		branch is up-to-date with its upstream tracking branch.
-		fmt.Println(defaultBranch)
+		if currentBranch == defaultBranch {
+			_, _ = fmt.Fprint(os.Stderr,
+				colors.Success("* "), colors.Success(defaultBranch), "\n",
+			)
+		} else {
+			fmt.Println(defaultBranch)
+		}
 		for branch, branchMeta := range branches {
 			if branchMeta.Parent != "" {
 				// not a stack root
 				continue
 			}
-			printStackTree(branches, branch, 1)
+			printStackTree(branches, repo, currentBranch, branch, 1)
 		}
 
 		return nil
 	},
 }
 
-func printStackTree(branches map[string]meta.Branch, root string, depth int) {
+func printStackTree(branches map[string]meta.Branch, repo *git.Repo, currentBranch string, root string, depth int) {
```

Comment:
nit: I think repo should be the first arg here (so it should be `repo, branches, currentBranch, root, depth` respectively).